test/integ/chunked-request.spec.js

```diff
@@ -78,13 +105,17 @@ describe('chunked-request', () => {
     });
   });
 
-  it('should catch errors raised by the chunkParser and pass them to the `onChunk` callback', () => {
+  it('should catch errors raised by the chunkParser and pass them to the `onChunk` callback', done => {
     const receivedChunks = [];
     const onComplete = () => {
       const chunkErrors = receivedChunks.filter(v => v instanceof Error);
-      expect(chunkErrors.length).toBe(1, 'one error caught');
+      expect(chunkErrors.length).toBe(2, 'two errors caught');
```

Comment:
This appears to be failing the integration tests; I'm not sure I follow why there would be 2 errors - pretty sure there is only one (certainly looks that way from the failed tests on IE and firefox - not sure why the Chrome ones didn't start).